src/services/api/search/v2/extract/search.extract.service.ts

```diff
@@ -32,6 +30,9 @@ const TYPE_TO_INDEX = (
   [SearchEntityTypes.POST]: `${indexPattern}posts`,
   [SearchEntityTypes.USER]: `${indexPattern}users`,
   [SearchEntityTypes.ORGANIZATION]: `${indexPattern}organizations`,
+  [SearchEntityTypes.CALLOUT]: `${indexPattern}callouts`,
+  [SearchEntityTypes.GROUP]: '',
```

Comment:
Is this needed?

Reply:
Yes. There were a couple of duplicate types, so I've created one complete type.
The group is still needed in search v1.